packages/kbn-test/src/functional_tests/tasks.ts

```diff
@@ -109,7 +109,9 @@ export async function runTests(options: RunTestsParams) {
 
         let es;
         try {
-          es = await runElasticsearch({ config, options: { ...options, log } });
+          if (process.env.DONT_START_ES !== 'true') {
```

Comment:
We're using some similar environment variables with the format `TEST_ES_`. How about changing this to `TEST_ES_DISABLE_STARTUP`

Reply:
Totally agree 👍🏼